hack/clean-up.sh

```diff
@@ -2,7 +2,7 @@
 oc patch nodeobservability/nodeobservability-sample --patch '{"metadata":{"finalizers": []}}' --type=merge
 oc delete nodeobservability nodeobservability-sample
 oc delete scc node-observability-scc
-oc delete clusterrole node-observability-cr
-oc delete clusterrolebinding node-observability-crb
+oc delete clusterrole node-observability-controller-role
+oc delete clusterrolebinding node-observability-controller-role-binding
```

Comment:
```suggestion
oc delete clusterrole node-observability-operator-agent
oc delete clusterrolebinding node-observability-agent
```

Reply:
I think we can delete this script, seems unused.